docs/release-notes/rl-0.8.md

```diff
@@ -301,6 +301,7 @@
 - Update default `telescope.setupOpts.pickers.find_files.find_command` to only
   include files (and therefore exclude directories from results)
 
-  [ckoehler](https://github.com/ckoehler):
+[ckoehler](https://github.com/ckoehler):
 
-  - Fix oil config referencing snacks
+- Fix oil config referencing snacks
+- Add `flash` plugin to `vim.utility.motion.flash`
```

Comment:
Could you include a link to the plugin page here? There are examples for non-inline markdown references above in this document.